airflow/jobs/scheduler_job_runner.py

```diff
@@ -850,7 +850,9 @@ def process_executor_events(
                 ti.task = task
                 if task.on_retry_callback or task.on_failure_callback:
                     request = TaskCallbackRequest(
-                        full_filepath=ti.dag_model.fileloc,
+                        filepath=str(ti.dag_model.fileloc),
+                        bundle_name=ti.dag_model.bundle_name,
+                        bundle_version=ti.dag_model.bundle_version,
```

Comment:
This should be the from dagrun too.

Reply:
I made this to be from the dagversion which I think should be more authentic